src/capabilities.ts

```diff
@@ -12,6 +12,7 @@ export type CapabilityMode = 'full' | 'normative' | 'terminology';
 export interface CapabilitiesRequest {
     fhirVersion: FhirVersion;
     mode: CapabilityMode;
+    hostName?: string;
```

Comment:
I suggest renaming this to `fhirServiceBaseUrl`. See my comment on https://github.com/awslabs/fhir-works-on-aws-routing/pull/153#discussion_r803214392

Reply:
I updated CapabilitiesRequest.hostname? to CapabilitiesRequest.fhirServiceBaseUrl